packages/block-library/src/navigation/edit/leaf-more-menu.js

```diff
@@ -146,7 +146,6 @@ export default function LeafMoreMenu( props ) {
 						<AddSubmenuItem
 							block={ block }
 							onClose={ onClose }
-							expanded
```

Comment:
The expanded prop was being passed to AddSubmenuItem but never actually used in the component. The function only uses expandedState and expand props for its logic. This was a cleanup to remove dead code that was likely leftover from an earlier implementation.